scripts/HOWTO-distribution.md

```diff
@@ -90,22 +90,19 @@ This is the next release in the 4.24 cycle. It's intended to be created from the
 
 - Update the help/en/Acknowledgements.shtml help page with any recent changes
 
-- Remake the help map, search, index and TOC. Be patient, it might take a couple minutes. It will pop some frames, etc, but should be entirely automatic. I/O exceptions in Sidebar* files are normal and expected.
+- Remake the JmriHelp control files, webindex.shtml and webtoc.shtml.
 
 ```
-        cd help/en/
-        ant
-        cd ../..
-
+        ant buildhelp
```

Comment:
Does this work from the main JMRI directory (i.e. help/..)? Or is it from help/en?

Reply:
> Does this work from the main JMRI directory (i.e. help/..)? Or is it from help/en?

`buidlhelp` is part of the main build.xml along with all of the other targets.  It invokes the language specific ant commands to create the JmriHelp control files.

Reply:
Thanks